### ISSUE TITLE
src/gui is not available

### ISSUE THREAD

**Issue by IntelOrca** [Sadret/openrct2-scenery-manager#28]
The GUI part of the plugin doesn't seem to be on GitHub. This prevents the plugin from being built by anyone else or CI.

**Comment by Sadret** [comment]
Yes, some time ago I moved the gui code to a separate project because I wanted to use it in other projects aswell. Since then, I have not decided yet how to publish it.

I thought about making it a separate plugin, so that the code (which is quite a lot) does not have to be loaded multiple times if multiple plugins use it. (Afaik this is possible in DUK with global variables?) On the other hand, as there is currently no dependency system, this would mean more work for the users.

So I should probably just upload the code and add building instructions in the readme.

**Comment by IntelOrca** [comment]
You could try using a submodule if that makes things easier.
